### PR TITLE
feat(onboarding): Add team sostrades

### PR DESCRIPTION
**Pull request by sostrades-dpeltre**
Pull request following the guide 'How to onboard a new project to a cluster on Operate First Cloud' to integrate SosTrades Project in Operate-first.

### CODE REVIEW

**Comment by HumairAK** [comment]
@sostrades-dpeltre can describe what this project is about? 

@erikerlandson @caldeirav @redmikhail : are we handing out access to ocp console/namespaces for the osc-cl1/osc-cl2 clusters?

**Review by erikerlandson — CHANGES_REQUESTED** [review]

File: cluster-scope/overlays/prod/osc/osc-cl1/kustomization.yaml [hunk 1]
```diff
@@ -1,43 +1,41 @@
----
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
-
-resources:
```
[the remark this answers is not in the document]
Comment (HumairAK) [reply cluster-scope/overlays/prod/osc/osc-cl1/kustomization.yaml:5]: this is an automated process that is reformatting the file (probably someone manually changed it and the automated tool is changing it back) it's expected

**Review by erikerlandson — APPROVED** [review]

**Comment by erikerlandson** [comment]
@HumairAK   we are handing out namespaces to projects that need to run services, on a case by case basis.  I am unfamiliar with this particular project, so @caldeirav or @MichaelTiemannOSC  might know if they approved this.

**Comment by MichaelTiemannOSC** [comment]
sos-trades is a 6-repo contribution fromt Airbus for their "System of Systems" (SOS) platform.  See https://github.com/os-climate/sostrades-core

This is one of two key namespaces needed for Airbus (the other being WITNESS).

**Comment by sostrades-dpeltre** [comment]
Thank you @HumairAK for accepting the pull request.
Here is a description of the project : SoSTrades is the tool used for the implementation of WITNESS or Transition Tool within OS-Climate. We plan to connect with Pysical risk and Data commons in the near future. SoSTrades is a web-based, multi-user, interactive publication-quality graph simulation platform. It allows users to drop new modules without additional coding, and provides embedded advanced numerical capabilities for simulation and multi-disciplinary optimization. It also has built-in collaborative capabilities to allow different experts to work together.

**Pull request merged**